guides/common/modules/con_prerequisites-for-cloning.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="prerequisites-for-cloning"]
+= Prerequisites for cloning
+
+To clone {ProjectServer}, ensure that you have the following resources available:
+
+* A minimal install of {EL} 9 to become the target server.
+Do not install {EL} 9 software groups or third-party applications.
+Ensure that your server complies with all the required specifications.
+For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_.
```

Comment:
out of scope: This link is broken but used three times already in HEAD of "master". I think it's worth fixing this in a separate follow-up PR shortly afterwards.